packages/core/src/components/HeaderIconLinkRow/IconLinkVertical.tsx

```diff
@@ -13,21 +13,22 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import * as React from 'react';
+import React from 'react';
 import classnames from 'classnames';
 import { makeStyles, Link } from '@material-ui/core';
 import LinkIcon from '@material-ui/icons/Link';
-import { Link as RouterLink } from 'react-router-dom';
+import { Link as RouterLink } from '../Link';
 
-export type IconLinkVerticalProps = {
+type IconLinkVerticalProps = {
   icon?: React.ReactNode;
   href?: string;
   disabled?: boolean;
   title?: string;
```

Comment:
This title doesn't seem to be used much, just passed on to MUI Link which afaik doesn't use it. Could perhaps get rid of it?